src/main/groovy/jenkins/automation/builders/PipelineJobBuilder.groovy

```diff
@@ -0,0 +1,84 @@
+package jenkins.automation.builders
+
+import javaposse.jobdsl.dsl.DslFactory
+import javaposse.jobdsl.dsl.Job
+import jenkins.automation.utils.CommonUtils
+
+/**
+ * The basic PipelineJob builder
+ *
+ * @param stages list of maps describing the stages; see example linked below for the shape of the map
+ * @param name job name
+ * @param description job description
+ * @param pipelineScript optional string containing the raw pipeline script (use instead of `stages`)
+ * @param pollScmSchedule optional string in cron format to trigger builds on a scheduled interval
+ * @param emails list of email addresses to receive notifications
+ * @see <a href="https://github.com/cfpb/jenkins-automation/blob/gh-pages/docs/examples.md#pipeline-builder" target="_blank">Pipeline job builder example</a>
+ */
+
+class PipelineJobBuilder {
+    List stages
+    String name
+    String description
+    String pipelineScript = ""
+    String pollScmSchedule
+    List<String> emails
```

Comment:
emails / extended email aren't actually an option on pipelines, so you'll need to remove this and the extended email block below